index.html

```diff
@@ -66,6 +66,7 @@ <h3 class="headlineh2">I am a Software Developer</h3>
             </li>
         </ul>
     </section>
+    <div class="project-popup-mobile popup"></div>
     <section class="works" id="works">
         <div class="works-header">
```

Comment:
https://github.com/lucas-crodrigues/lucas-crodrigues.github.io/blob/28da9e77263dea377709775a3bf9fb7ce5589e24/index.html#L75-L245

- [ OPTIONAL] In this particular project, the whole projects section along with the modals should be created dynamically with Javascript as you did with you modals popup.
- The modals  is already done (good job!), but the works projects section are still not being created with Javascript.  Kindly please use Javascript to dynamically render your works projects section.

You could reuse the code that you already have.

- That way, adding projects in the future will be easy. You will only need to add it in one place - as another object in the `data` array.

![image](https://user-images.githubusercontent.com/57421823/152509546-f41b40b6-dc88-4e3f-9ab4-7bee886e1f52.png)